sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
+  if array == nil || array.length <= 1
+    return array # nothing to sort
+  end
+
+  index = 0
+  swapped = true
+  while index < array.length && swapped
```

Comment:
Are you sure this "swapped" guard is working as corrected? I believe it could exit this loop prematurely if array[index] and array[other_index] didn't need to be swapped. Have you tested this loop?